data/rulesets.js

```diff
@@ -535,6 +535,20 @@ let BattleFormats = {
 			this.add('rule', 'Z-Move Clause: Z-Moves are banned');
 		},
 	},
+	nfeclause: {
+		effectType: 'ValidatorRule',
+		name: 'NFE Clause',
+		desc: "Bans Pok&eacute;mon that cannot evolve",
```

Comment:
I feel like "cannot evolve" is a bit vague. I would say either "Bans Pokemon that are fully evolved", or "Bans Pokemon that are fully evolved or can't evolve".

Reply:
Agree. Should be the latter.

Reply:
Should be good now